app/mailers/alert_notifier.rb

```diff
@@ -3,9 +3,10 @@ class AlertNotifier < ActionMailer::Base
   helper :application, :applications
 
   def alert(theme, alert, applications, comments = [])
-    @alert, @applications, @comments = alert, applications, comments
-    # TODO: Differentiate people with expired subscriptions
-    @trial_subscriber_analytics_params = {utm_source: "alert", utm_medium: "email", utm_campaign: "trial_subscriber"}
+    @theme = theme
+    @alert = alert
+    @applications = applications
+    @comments = comments
```

Comment:
I split these apart in https://github.com/openaustralia/planningalerts/commit/be39ee3eb561a8b065b89da2abe1f8412a96d08d 

It's less concise, but I think more readable to have the assignment right next to the term, rather than look back and forth along the line. Ruby style guide: https://github.com/bbatsov/ruby-style-guide#parallel-assignment

Happy to leave this as is?